internal/logger/target/kafka/kafka.go

```diff
@@ -197,6 +197,10 @@ func (h *Target) Init() error {
 	return nil
 }
 
+// Cancel - cancels the target
+func (h *Target) Cancel() {
+}
+
```

Comment:
Leaving this empty potentially causes problems, you may have to refactor the Kafka part as well since you are making the entire logger subsystem dynamic.

Reply:
This function is currently not called at all, so should not cause problems. But it is required as it is being added to the `Target` interface. Code can be added to it when making the audit logging config dynamic. Can be done in a separate PR.